### ISSUE TITLE
Random timeout issue with requests

### ISSUE THREAD

**Issue by driverjb** [duosecurity/duo_api_nodejs#19]
The IP address in the error resolves to the Duo main site. No where do I reference that in my code (my unique admin api host is in a configuration file).

```text
Error: connect ETIMEDOUT 52.32.63.128:443
    at Object._errnoException (util.js:992:11)
    at _exceptionWithHostPort (util.js:1014:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1186:14)
```

All of the calls being used in my app have been working for almost a year without issue. Over this weekend I keep seeing this error without any real details other than what you see above. The code requires a callback and does not provide any sort of error handler for when the api cannot be contacted.

**Comment by driverjb** [comment]
I skimmed through the code in the repo and it looks like you need to include:

```javascript
req.on('error', (err) => {
  callback(err)
})
```

Since the provided API expects the first parameter to be the response (breaking standard callback syntax of error first, everything else after) you would probably need to up the version to 2.0 (breaking everything that depends on this). Or you could go non-standard and have the error tacked onto the end:

```javascript
req.on('error', (err) => {
  callback(null, err)
})
```

**Comment by vbscott** [comment]
Solved with latest pull request (#21).
